libtextsecure/protobufs.js

```diff
@@ -4,15 +4,11 @@
     window.textsecure.protobuf = {};
 
     function loadProtoBufs(filename) {
-        return dcodeIO.ProtoBuf.loadProtoFile({root: 'protos', file: filename}, function(error, result) {
-           if (error) {
-              throw error;
-           }
-           var protos = result.build('textsecure');
-           for (var protoName in protos) {
-              textsecure.protobuf[protoName] = protos[protoName];
-           }
-        });
+        var result = dcodeIO.ProtoBuf.loadProtoFile({root: 'protos', file: filename});
```

Comment:
No, I think we can revert it. It was a test to see if it was a file-loading race condition on windows. Will move to another branch for any continued exploration like that. :0/